src/components/Book.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+
+export default function Book({ book }) {
+  return (
+    <div className="bookInfo">
+      <p className="genre">{book.genre}</p>
```

Comment:
- [ ] Please, consider rendering only the title and author, as required in project 👍